nixos/modules/services/web-apps/engelsystem.nix

```diff
@@ -0,0 +1,654 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib) mkDefault mkEnableOption mkIf mkOption types;
+
+  cfg = config.services.engelsystem;
+  fpm = config.services.phpfpm.pools.engelsystem;
+
+  user = "engelsystem";
+  package = pkgs.engelsystem;
+
+  engelsystemConfig = pkgs.writeTextFile {
+    name = "config.php";
+    text = ''
+      <?php
+      return [
```

Comment:
Do you think we could just parse in json? That would be a lot easier to generate and has less problems with escaping. Also this would allow to have settings set without having an explicit option.

Reply:
i.e. https://github.com/NixOS/nixpkgs/pull/75584

Reply:
You mean to parse the json from the php? I'm not good enough in PHP to do that

Reply:
@Kloenk see https://github.com/NixOS/nixpkgs/blob/216cd6c5a30af9018d59556f549af9e1c50bdb0a/nixos/modules/services/web-apps/limesurvey.nix#L21

Reply:
I built something like this in my branch feature/engelsystem2, feel free to adapt the changes and include them here. I also used utils.genJqSecretsReplacementSnippet to add in the secrets.

Reply:
adapted the json config, now will try to do the mysql connection